src/stories/CheckboxList.stories.jsx

```diff
@@ -0,0 +1,30 @@
+import CheckboxList from '~/components/checkbox-list/CheckboxList'
+
+export default {
+  title: 'CheckboxList',
+  component: CheckboxList,
+  argTypes: {
+    items: [],
+    title: {
+      type: 'string',

```

Comment:
use ts and import interface from your component so you don't need write type manualy

Reply:
thx but as i know we dont need to write stories in ts

Reply:
it makes stories more informative, so it will be better if we will use it

Reply:
agree with you